packages/ui-extensions-sdk/src/events/events.ts

```diff
@@ -49,6 +50,13 @@ interface DDEventDataTypes<AuthStateArgs> {
     [EventType.API_ACCESS_CHANGE]: APIAccessChangeEvent;
 }
 
+const deprecationWarnings: Partial<Record<EventType, string>> = {
+    [EventType.DASHBOARD_TIMEFRAME_CHANGE]:
+        'The "dashboard_timeframe_change" event is deprecated. You may subcribe to updated dashboard timeframes from the more general "context_change" event.',
+    [EventType.DASHBOARD_TEMPLATE_VAR_CHANGE]:
+        'The "dashboard_template_var_change" event is deprecated. You may subscribe to template variable changes from the more general "context_change" event.'
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏I think this is going to create a bunch of warnings for the `useTemplateVariable` hook since it [listens to this event](https://github.com/DataDog/apps/blob/a4ebbdb28f09ae4ff7a41a3a5a630fc462fdc8c6/packages/ui-extensions-react/src/use-template-variable.ts#L75). We should be able to drop that event listener though and not worry about it, based on the motivation behind this change.

Reply:
oh good catch. I'll update that hook to instead use `context`